packages/oauth/src/index.ts

```diff
@@ -495,7 +508,7 @@ export interface CallbackOptions {
   // callbackRes.
   success?: (
     installation: Installation | OrgInstallation,
-    options: InstallURLOptions,
+    options: InstallURLOptions | undefined,
```

Comment:
Anyone see any issues will allowing options here to be undefined? It appears in the built-in success and failure callbacks these options are referenced but not used.

Reply:
This should be fine but changing the type can be a breaking change for existing TypeScript app maintainers. Instead of passing `undefined`, passing an empty `InstallURLOptions` object may be an alternative. This PR does not change the existing app behavior. No existing apps don't receive the empty value.

Reply:
That makes sense, I've updated this to pass an empty `InstallURLOptions` object instead of passing undefined.